scripts/buildBranchAPI.js

```diff
@@ -102,6 +106,11 @@ async function build() {
 
   // Copy necessary code and configuration over
   fs.copySync(path.join(srcDir, 'packages', '@adobe', 'spectrum-css-temp'), path.join(dir, 'packages', '@adobe', 'spectrum-css-temp'));
+  try {
```

Comment:
note, this is only needed temporarily, once we do a release, we can delete this code